internal/oem/oem.go

```diff
@@ -238,6 +239,34 @@ alias gsutil="(docker images google/cloud-sdk || docker pull google/cloud-sdk) >
 		name:  "file",
 		fetch: file.FetchConfig,
 	})
+	configs.Register(Config{
+		name:  "sakuracloud",
+		fetch: sakuracloud.FetchConfig,
+		baseConfig: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.SystemdUnit{
+					{
+						Enable: true,
+						Name:   "coreos-metadata-sshkeys@.service",
+					},
+					{
+						Enable: true,
+						Name:   "coreos-sakuracloud-network.service",
```

Comment:
I think this service file will be written by ignition on "files" stage. See https://github.com/coreos/ignition/blob/master/internal/exec/stages/files/files.go#L274